composables/useListInfiniteScroll.ts

```diff
@@ -35,7 +35,7 @@ export default function ({
   const scrollItemHeight = ref(300)
   const itemsPerRow = ref(4)
   const scrollItemSizeInit = ref(false)
-  const first = ref(defaultFirst || 20)
+  const first = ref(defaultFirst || 40)
```

Comment:
Isn't this causing it too slow? 

Loading 40 items takes quuite a memory

Reply:
I have discovered that the time required to fetch 20 items versus fetching 40 items is quite similar. Both of them take 500ms or so. Or I would limit it to 30. WDYT

Reply:
Ok if it's like that I am fine. 
Or let's do 30 and we will see